source/gui/settingsDialogs.py

```diff
@@ -1948,6 +1965,11 @@ class KeyboardSettingsPanel(SettingsPanel):
 	# Translators: This is the label for the keyboard settings panel.
 	title = _("Keyboard")
 	helpId = "KeyboardSettings"
+	panelDescription = _(
+		# Translators: This is a label appearing on the keyboard settings panel.
+		"Configures the various settings that influence keyboard behaviour."
+		" These settings also control how speech output is treated when typing.",
```

Comment:
Could you please reformat these strings so that the space is on the previous line, e.g.

```suggestion
		"Configures the various settings that influence keyboard behaviour. "
		"These settings also control how speech output is treated when typing. ",
```